app/assets/javascripts/discourse/app/components/quote-button.js

```diff
@@ -325,6 +234,22 @@ export default Component.extend(KeyEnterEscape, {
     });
   },
 
+  _setupSelectionListeners() {
+    const updateRect = () => this.textRange.updateRect();
+    document.querySelector(".cooked").addEventListener("mouseup", updateRect);
```

Comment:
```suggestion
    document.querySelector(".cooked").addEventListener("mouseup", updateRect);
```

I'm not sure `document.querySelector(".cooked")` is the best target for the mouseup event. Since we're only showing one quote button at a time, maybe we can attach this event to the document body? 

Another thing I noticed here when pulling this locally is that after user selects text, the event listeners stick around even after the quote button component is no longer visible (they do get cleared if you navigate away from the topic route). Can we maybe remove them when the popper is dismissed/hidden as well? 

Related to the listeners, I'm seeing an error locally: 

```
virtual-element-from-text-range.js:16 Uncaught TypeError: this.range.getBoundingClientRect is not a function
    at VirtualElementFromTextRange.updateRect (virtual-element-from-text-range.js:16:1)
    at updateRect (quote-button.js:257:1)
```

Repro steps: select some text, see the quote button component, then click outside it so it is now hidden, and then scroll the page.